internal/daemon/daemon.go

```diff
@@ -59,18 +59,18 @@ type Daemon struct {
 	hooks config.Hooks // Hooks to be called upon various daemon actions.
 
 	ReadyChan      chan struct{}      // Closed when the daemon is fully ready.
-	ShutdownCtx    context.Context    // Cancelled when shutdown starts.
-	ShutdownDoneCh chan error         // Receives the result of the d.Stop() function and tells the daemon to end.
-	ShutdownCancel context.CancelFunc // Cancels the shutdownCtx to indicate shutdown starting.
+	shutdownCtx    context.Context    // Cancelled when shutdown starts.
+	shutdownDoneCh chan error         // Receives the result of the d.Stop() function and tells the daemon to end.
```

Comment:
Comment mentions d.Stop that is now d.stop

Reply:
Not addressed

Reply:
I added it to the commit which introduces the `exit()` function as that's the only place this is actually used now.

Reply:
I've updated it in both commits now.